src/components/Hero/Hero.tsx

```diff
@@ -160,7 +160,14 @@ export const Hero = chakra(
       const childrenToRender =
         heroType === "campaign" ? (
           <>
-            <Image alt={imageProps.alt} src={imageProps.src} />
+            <Box
+              __css={styles.imgWrapper}
+              style={{
```

Comment:
Would this work if you merge `styles.imgWrapper` with `{ backgroundImage: ... }`. Just to have one prop instead of both `__css` and `style`, but not a big deal.

Reply:
Like this?
```
__css={{
  ...styles.imgWrapper,
  backgroundImage: `url(${imageProps.src})`,
}}
```